app/waf/src/models/adc.model.ts

```diff
@@ -191,14 +177,3 @@ export class ActionsResponse extends Entity {
   id: string;
 }
 
```

Comment:
need to delete  empty line at the end of this file. otherwise it won't pass tests.  after running "npm run prettier:fix", you should add those modifications by 'git add XXXX, git commit --amend, etc'.